File: coin-jrpc-client/src/main/java/com/waykichain/coin/wicc/po/DecodeRawTxPO.java
package com.waykichain.coin.wicc.po;

import lombok.Data;

/**
 * @Author: yanjunlin
 * @CreateDate: 2019/4/13 15:45
 * @Description: 根据签名字段解析原始交易单
 */
@Data
public class DecodeRawTxPO {

    private String hexstring;
}

File: wicc-chain-pojo/src/main/kotlin/com/waykichain/chain/po/v2/DecodeRawtxPO.kt
package com.waykichain.chain.po.v2

import io.swagger.annotations.ApiModelProperty
import org.hibernate.validator.constraints.NotBlank

/**
 *
 * @Author:         yanjunlin
 *
 * @CreateDate:     2019/4/13 15:55
 *
 * @Description:    根据签名字段解析原始交易单
 *
 */
class DecodeRawtxPO {

    @ApiModelProperty("签名后的交易字段")
    @NotBlank(message = "[in param error] hexstring  is null")
    var rawtx: String? = null
}